docs/authentication.rst

```diff
@@ -65,8 +75,45 @@ Here is an example::
       });
     });
 
-Your API clients can then use this key to authenticate against your API.  This
-is how you would protect the secret endpoint with basic authentication::
+This is a naive example which simply prints out the API Keys for them, but
+once they have the keys they will be able to authenticate with your API.
+
+For more information on API Keys, please see
+`Using Stormpath for API Authentication`_
+
+Enabling API Authentication
+---------------------------
+
+Regardless of the authentication strategy that you choose, you will need to
```

Comment:
Mmmm. This isn't true. Basic Auth will work without this option being turned on. This only hooks up the oauth2 routes.